src/api/c/pinverse.cpp

```diff
@@ -0,0 +1,170 @@
+/*******************************************************
+ * Copyright (c) 2014, ArrayFire
```

Comment:
This is a new file so you should set this to 2018

Reply:
Makes sense, I just changed this one and test/pinverse.cpp